### PR TITLE
add unit test for documents

### LINKED ISSUE #10: Add "import" unit test

### PR DESCRIPTION
**Pull request by tfe2012**
Closes #10

### CODE REVIEW

File: Tests/Unit/Document/AbstractGetterSetterTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            $this->assertEquals(3, count($this->document->$getter()));
+        }
+    }
+
+    /**
+     * Attributes test data provider.
+     *
+     * @return array
+     */
+    abstract public function attributesDataProvider();
```
Comment (tautrimas) [inline Tests/Unit/Document/AbstractGetterSetterTest.php]: Standard requires providers to be above their tests.

File: Tests/Unit/Document/AbstractGetterSetterTest.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\MagentoConnectorBundle\Tests\Unit\Document;
+
+use ONGR\MagentoConnectorBundle\Tests\app\fixtures\ExpectedDocuments\ExpectedDocuments;
+
+abstract class AbstractGetterSetterTest extends \PHPUnit_Framework_TestCase
```
Comment (tautrimas) [inline Tests/Unit/Document/AbstractGetterSetterTest.php:16]: Does https://github.com/ongr-io/ConnectionsBundle/blob/8d1274010112fec7653cdaf78eef3d8bf3e9eb03/Tests/Unit/Entity/AbstractEntityTest.php do the same? Maybe register issue and leave as is currently?

**Comment by tautrimas** [comment]
- [x] Replace all `Set up document for testing.` and other duplicate comments with `{@inheritdoc}`.

File: Tests/Unit/Document/AbstractGetterSetterTest.php (hunk 2 above)
Comment (tfe2012) [inline Tests/Unit/Document/AbstractGetterSetterTest.php:16]: https://github.com/ongr-io/MagentoConnectorBundle/issues/27

File: Tests/Unit/Document/CategoryDocumentTest.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+     */
+    protected function setUp()
+    {
+        $this->document = new CategoryDocument();
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
```
Comment (tautrimas) [inline Tests/Unit/Document/CategoryDocumentTest.php]: No need to add `@return` with `{@inheritdoc}`.

File: Tests/Unit/Document/CategoryObjectTest.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    protected function setUp()
+    {
+        $this->document = new CategoryObject();
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
```
Comment (tautrimas) [inline Tests/Unit/Document/CategoryObjectTest.php]: No need to add `@return` with `{@inheritdoc}`.

File: Tests/Unit/Document/CdnObjectTest.php [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+     */
+    protected function setUp()
+    {
+        $this->document = new CdnObject();
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
```
Comment (tautrimas) [inline Tests/Unit/Document/CdnObjectTest.php]: No need to add `@return` with `{@inheritdoc}`.

**Comment by tautrimas** [comment]
- [x] No need to add `@return` with `{@inheritdoc}`. `@return` can be removed.

**Pull request merged**